src/main/java/spoon/reflect/cu/CompilationUnit.java

```diff
@@ -12,37 +12,4 @@
  * public type declaration and other secondary types declarations (not public).
  */
 public interface CompilationUnit extends CtCompilationUnit {
```

Comment:
I guess the whole interface should be put as deprecated now? or is it still used internally?

Reply:
good idea, done.